src/components/Grid.js

```diff
@@ -0,0 +1,37 @@
+import chota from "chota";
+import React from 'react';
+import Graph from './Graph'
```

Comment:
I would rather split this pull request into multiple, as this is not really a part of this issue (it is more related to #9).

Reply:
But if it's okay with everybody else (@RalphErkamps @EricWoude @Sebastiaanvm), it might be overkill for this relatively small pull request.